Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenView.cs

```diff
@@ -124,5 +124,23 @@ public async UniTask HideTipWithFadeAsync(int index, float duration, Cancellatio
 
             tips[index].gameObject.SetActive(false);
         }
+
+        public async UniTask ChangeBackgroundColor(Material material, string property, Color toColor, float duration, CancellationToken cancellationToken = default)
+        {
+            Color currentColor = material.GetColor(property);
+            float time = 0f;
+            while (time < duration)
+            {
+                if (cancellationToken.IsCancellationRequested)
```

Comment:
No need to check if cancelled, just do: `cancellationToken.ThrowIfCancellationRequested();`

Reply:
Changed